Gifski/DropVideoViewController.swift

```diff
@@ -0,0 +1,113 @@
+import AppKit
+import Crashlytics
+import AVKit
+
+final class DropVideoViewController: NSViewController {
+	private lazy var videoDropView = with(VideoDropView()) {
+		$0.frame.size = CGSize(width: 360, height: 240)
```

Comment:
I don't think this should be hard-coded.

Reply:
Somehow we need to specify a window size for the controller. I cannot use the size of a current window as this might be e.g. Edit controller that has a different size than most of the controllers. Maybe we would need a constant size for all the controllers except the Edit one?

Reply:
Yes, I would prefer defining the size in the view controller view instead of here.